game_eggs/steamcmd_servers/eco/egg-eco.json

```diff
@@ -77,6 +81,15 @@
             "user_viewable": true,
             "user_editable": false,
             "rules": "required|string|max:20"
+        },
+        {
+            "name": "Extra Flags",
```

Comment:
This isn't actually applied anywhere in the egg?

Reply:
It is. It's Parkers Steam DL Script.

See https://github.com/parkervcp/eggs/blob/77317bb22cc3de258b17639fa271d4db168e0167/game_eggs/steamcmd_servers/eco/egg-eco.json#L21